utilities/oadp.py

```
@@ -75,13 +135,24 @@ def __enter__(self) -> "VeleroBackup":
         return self
 
     def __exit__(self, exception_type, exception_value, traceback) -> None:
+        teardown_error = None
         try:
             if self.teardown:
-                delete_velero_resource(resource=self, client=self.client)
+                try:
+                    delete_velero_resource(resource=self, client=self.client)
+                except Exception as error:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)